psydac/fem/basic.py

```diff
@@ -72,7 +72,7 @@ def symbolic_space( self ):
     # Abstract interface: evaluation methods
     #---------------------------------------
     @abstractmethod
-    def eval_field( self, field, *eta , weights=None):
+    def eval_field( self, field, *eta, weights=None):
         """
         Evaluate field at location(s) eta.
```

Comment:
I've just realized that this abstract interface is probably not enforced across Psydac, as it should be. Can you please check that all subclasses of `FemSpace` accept the `weights` argument?

Reply:
done in 217f0ff